site/en/install/docker.md

```diff
@@ -37,7 +37,7 @@ using the following format:
 |-------------|----------------------------------------------------------------------------------------------------------------------|
 | `latest`    | The latest release of TensorFlow CPU binary image. Default.                                                          |
 | `nightly`   | Nightly builds of the TensorFlow image. (Unstable.)                                                                  |
-| *`version`* | Specify the *version* of the TensorFlow binary image, for example\: *2.1.0*                                          |
+| *`version`* | Specify the *version* of the TensorFlow binary image, for example\: *2.8.0*                                          |
```

Comment:
```suggestion
| *`version`* | Specify the *version* of the TensorFlow binary image, for example\: *2.8.3*                                          |
```

@MarkDaoust @markmcd @shilpakancharla Should we update the TAG names in the table (see https://hub.docker.com/r/tensorflow/tensorflow/tags)

Reply:
I see that em + inline code isn't rendering on the site - I've filed b/264824237 internally to fix it.

> Should we update the TAG names in the table

What are you proposing we change?